modules/updates-and-upgrades.adoc

```diff
@@ -11,5 +11,8 @@ The decision regarding the need for a Service update to the Central instance and
 Customers have no control over when a Central service update occurs. For more information, see link:https://www.redhat.com/licenses/Appendix_4_Red_Hat_Online_Services_20221213.pdf[PRODUCT APPENDIX 4 RED HAT ONLINE SERVICES].
 Upgrades to the version of {rh-rhacscs-first} are considered part of the service update.
 
-The version of {product-title-managed-short} used on Secured Clusters must match the version of the Central instance of {product-title-managed-short} to ensure compatibility.
-Customers are responsible for Secured Cluster services upgrades required to maintain this version compatibility.
\ No newline at end of file
+Customers are responsible for timely {product-title-short} Secured Cluster services upgrades required to maintain compatibility with {product-title-managed-short}.
+
+Red Hat recommends enabling automatic upgrades for Secured Clusters connected to  {product-title-managed-short}.
```

Comment:
```suggestion
Red Hat recommends enabling automatic upgrades for Secured Clusters that are connected to {product-title-managed-short}.
```
or
```suggestion
Red Hat recommends enabling automatic upgrades for Secured Clusters that connect to {product-title-managed-short}.
```

Reply:
I realize this is a weird distinction, but the clusters exist outside of RHACS and while they do connect to them, the component that manages the connection as a whole (Central) is within ACS so I think it's more clear to say "that are connected to", if that makes sense.